<?xml version="1.0" encoding="UTF-8"?>

<launch>
  <!-- Arguments -->
  <arg name="file_path" default="/home/$(env USER)/dynablox_output/clouds.csv" /> 

  <!-- Cloud Visualizer -->
  <node name="cloud_visualizer" pkg="dynablox_ros" type="cloud_visualizer" output="screen" args="--alsologtostderr" required="true">
    <param name="file_path" value="$(arg file_path)" />
    <param name="static_point_scale" value="0.05" />
    <param name="dynamic_point_scale" value="0.1" />
    <rosparam param="out_of_bounds_color">[0.8,0.8,0.8,1]</rosparam>
  </node>
  
  <!-- Visualize -->
  <node type="rviz" name="rviz" pkg="rviz" args="-d $(find dynablox_ros)/config/rviz/cloud_visualizer.rviz"/>

</launch>
